packages/fxa-admin-panel/src/components/PageAccountSearch/Account/index.tsx

```diff
@@ -0,0 +1,389 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { gql, useMutation } from '@apollo/client';
+import {
+  Account as AccountType,
+  SecurityEvents as SecurityEventsType,
+  Totp as TotpType,
+  RecoveryKeys as RecoveryKeysType,
+  LinkedAccount as LinkedAccountType,
+} from 'fxa-admin-server/src/graphql';
+import { AdminPanelFeature } from 'fxa-shared/guards';
+import Guard from '../../Guard';
+import Subscription from '../Subscription';
+import { ConnectedServices } from '../ConnectedServices';
+import { TableRowYHeader, TableYHeaders } from '../../TableYHeaders';
+import { TableRowXHeader, TableXHeaders } from '../../TableXHeaders';
+import EmailBounces from '../EmailBounces';
+import { getFormattedDate } from '../../../lib/utils';
+import DangerZone from '../DangerZone';
+import ResultBoolean from '../../ResultBoolean';
+import { HIDE_ROW } from '../../../../constants';
+
+export type AccountProps = AccountType & {
+  onCleared: () => void;
+  query: string;
+};
+
+export const RECORD_ADMIN_SECURITY_EVENT = gql`
+  mutation recordAdminSecurityEvent($uid: String!, $name: String!) {
+    recordAdminSecurityEvent(uid: $uid, name: $name)
+  }
+`;
+
+export const EDIT_LOCALE = gql`
+  mutation editLocale($uid: String!, $locale: String!) {
+    editLocale(uid: $uid, locale: $locale)
+  }
+`;
+
+export const UNLINK_ACCOUNT = gql`
+  mutation unlinkAccount($uid: String!) {
+    unlinkAccount(uid: $uid)
+  }
+`;
+
+export const LinkedAccount = ({
```

Comment:
There's even more refactoring we could do here, like `LinkedAccounts` and `AccountDetails` at least could probably use their own components and I'm not sure if we prefer all of our GQL queries/mutations to be in one file but this is a step in the right direction. 🤷‍♀️ Happy to make any further requested changes.